scan/lib/scan/test_command_generator.rb

```diff
@@ -35,8 +35,9 @@ def options
         options << destination # generated in `detect_values`
         options << "-derivedDataPath '#{config[:derived_data_path]}'" if config[:derived_data_path]
         options << "-resultBundlePath '#{result_bundle_path}'" if config[:result_bundle]
-        options << "-enableCodeCoverage YES" if config[:code_coverage]
-        options << "-enableAddressSanitizer YES" if config[:address_sanitizer]
+        options << "-enableCodeCoverage #{config[:code_coverage] ? 'YES' : 'NO'}" unless config[:code_coverage].nil?
```

Comment:
Not sure if we need a helper like `yesno` mixin for boolean type or `humanize`-like method...
So far `predicate ? 'YES' : 'NO'` seems to be the way to "stringify" booleans.